src/test/java/website/frontrow/GameTest.java

```diff
@@ -123,4 +125,111 @@ public void testTick2()
         game.tick();
         verify(levels.get(0), times(1)).tick();
     }
+    
+    /**
+     * Tests that a game over occurs when the player has no lives left.
+     */
+    @Test
+    public void testGameOverSetLives()
+    {
+
+    	levels.add(l);
+    	Player p = new Player(po);
+    	ArrayList<Player> list = new ArrayList<Player>();
+    	list.add(p);
+    	Game game = new Game(levels, list);
+    	game.start();
+    	assertTrue(game.isRunning());
+    	
+    	p.setLives(0);
+    	game.tickGO();
+    	assertFalse(game.isRunning());
+    	
```

Comment:
Only the whitespace at the bottom the one preceeding }, look at where I made the line comment :P